src/mappings/events/vaults.ts

```diff
@@ -55,6 +55,14 @@ export const vaultsEventKit = (block: any, data: any, module: string, path: stri
     vault.debt = payload?.debtSnapshot?.debt?.__value;
     vault.balance = payload?.locked?.__value;
     vault.state = payload?.vaultState;
+
+    if (vault.state === 'liquidating' && !vault.liquidationStartedAt) {
+      vault.liquidationStartedAt = block.block.header.time;
```

Comment:
While in the liquidating state, if there is a change in any other field, we will get a state_change event, and at that time, this if condition will be true and this will update the `liquidationStartedAt` time, which is not correct anymore. The `liquidationStartedAt` should only have time only first time this state changed into liquidation.

Same is true for `liquidatedAt`.